src/skeleton/types.js

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright (c) 2018-2020 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import type {OverrideT} from '../helpers/overrides.js';
+
+export type OverridesT = {
+  Row?: OverrideT,
+  Root?: OverrideT,
+};
+
+export type SkeletonPropsT = {
+  overrides?: OverridesT,
+  /** Defines the number of row element in a skeleton */
+  rows: number,
+  /** Deinfes if the skeleton has an animation default is false*/
+  animation: boolean,
+  /** Deinfes the height of the skeleton container*/
+  height?: string,
```

Comment:
Are `width` and `height` necessary? To me it seems like this would be a better fit for `className`.

```
const skeletonStyles = {width: '100px'}

const FooSkeleton = styled(Skeleton, skeletonStyles)
 // or alternatively
<Skeleton className={css(skeletonStyles)} />
```

Reply:
Because the height and width are the most important attribute for skeleton (the one who uses it will need to set it). I prefer to just leave them there and developers could use it conveniently.